main.py

```diff
@@ -262,6 +264,8 @@ def index():
     match graph.lower():
```

Comment:
`match` cannot be used with older python, so please change it if possible.
See #84

Reply:
changed

Reply:
Thanks!!